src/libraries/pkg/descriptions.json

```diff
@@ -126,6 +126,11 @@
     "Description": "File system globbing to find files matching a specified pattern.",
     "CommonTypes": []
   },
+  {
+    "Name": "Microsoft.Extensions.HostFactoryResolver.Sources",
+    "Description": "Internal package for sharing Microsoft.Extensions.Hosting.HostFactoryResolver type.",
```

Comment:
In case we ever plan to put this on NuGet, we may want to consider adding a note here saying that the package is not really supported

Reply:
That's what I meant by internal.  I'm already doing more than the original code from extensions, which just used the PackageID as the description.